src/components/PropertyGroupDropdown.tsx

```diff
@@ -7,21 +9,25 @@ type DropdownProps = {
 
 export function PropertyGroupDropdown (props: DropdownProps) {
     const possiblePropertyGroups = [
-        'All',
-        'Retail',
-        'Industrial',
-        'Commercial'
+        { value: 'All', label: 'All' },
+        { value: 'Retail', label: 'Retail' },
+        { value: 'Industrial', label: 'Industrial' },
+        { value: 'Commercial', label: 'Commercial' }
     ]
 
     return (
-        <label className="six dropdown">Property Group (Portals) <span className='downArrow'> ⬇ </span>
-            <select multiple value={props.chosenPropertyGroups} className="dropdown-content" id="#propertyGroup" onChange={(e: ChangeEvent<HTMLSelectElement>) => {
-                props.changePropertyGroups([...e.target.selectedOptions].map((option) => option.value))
-            }} tabIndex={3} >
-                {possiblePropertyGroups.map((propertyGroup) => {
-                    return <option key={propertyGroup} value={propertyGroup}>{propertyGroup}</option>
-                })}
-            </select>
+        <label className="six">Property Group (Portals)
+            <Select
+                getValue={props.chosenPropertyGroups}
+                id="#propertyGroup"
+                isMulti
+                onChange={(e: PropertyTypes[]) => {
+                    e = e === null ? [{ value: 'All', label: 'All' }] : e
```

Comment:
If there was only one value in the field and it was removed a null value would be returned which would cause an error b/c map isn't a method on null. This is to prevent this crash but to also make sure the default value for this field will also be 'All' if all of the values are removed.

Reply:
it's generally recommended not to change the value of parameters. 

Perhaps:
```
const propertyGroupValues = newPropertyGroups?.map((option: ReactSelectOption) => option.value) || ['All']
props.changePropertyGroups(propertyGroupValues)
```